wdl-lint/src/util.rs

```diff
@@ -1,5 +1,16 @@
 //! A module for utility functions for the lint rules.
 
+use wdl_ast::{AstToken, Comment, SyntaxKind};
+
+/// Detect is a comment is in-line or not by looking for `\n` in the prior
+/// whitespace. Used in CommentWhitespace and MalformedLintDirective rules.
```

Comment:
```suggestion
/// Detect if a comment is in-line or not by looking for `\n` in the prior
/// whitespace.
```
Tip: when writing comments (or really writing code in general) try and think about whether what you're writing is going to be easy to maintain.

e.g. this sentence I cut out, I removed it because I don't want to keep this updated if I ever start using the function in some third or fourth rule and I also don't want the comment to be incorrect. Best just not to say anything in this case.

Reply:
Makes sense, and sorry for the typo!